docs/dev/tags.rst

```diff
@@ -0,0 +1,27 @@
+
+
+Basic process of adding and using a new tag:
+--------------------------------------------
+
+To create a *global tag* (one applies to all programs), add a new line to the global tag dictionary ``all_global_tags`` in the
+file https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/tagdict/__init__.py.
+To create a *program-specific tag*, add the new tag's name to the program tag dictionary ``all_program_tags`` in the same file as above.
+Program tags are used with ``Tag.getProgramTag()`` or ``Tag.getBooleanTag()`` with a program argument in place of a boolean one in the
+first coordinate.
```

Comment:
I thought it meant you can pass a program as the first argument to `getBooleanTag`, but I did a bad thing and copied blindly from [https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/tagdict/\_\_init\_\_.py#L60](https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/tagdict/__init__.py#L60) without giving it too much thought. If it doesn't actually make any sense, I'm perfectly happy to remove it from this commit (and possibly the other file as well?).